__tests__/news.test.js

```diff
@@ -106,7 +102,77 @@ describe('GET /api/articles/:article_id', () => {
             expect(responseArray).toEqual("Not Found");
           })
     });
+    test('should return appropriate error if invalid input data type', () => {
+        return request(app)
+        .get(`/api/articles/TEXT`)
+        .expect(400)
+        .then((response) => {
+            const responseArray = response.res.statusMessage
+            expect(responseArray).toEqual("Bad Request");
+          })
+    });
+});
+
+describe('GET /api/articles', () => {
+    test('should return correct properties', () => {
+        return request(app)
+        .get(`/api/articles`)
+        .expect(200)
+        .then((response) => {
+            const responseArray = response.body.article
```

Comment:
Make sure that, when we do a forEach, we also do an array length check to avoid false positives. The for each will give a pass if the array is empty because nothing fails the assertions.